{{cookiecutter.repository_name}}/pyproject.toml

```diff
@@ -43,6 +41,7 @@ notebook = "5.7.8"
 jupyter_contrib_nbextensions = "0.5.1"
 ipykernel = "^5.4.3"
 ipywidgets = "^7.6.3"
+cleannb = { git = "https://github.com/KwatME/CleanNB.py.git", rev = "8abfa6ad" }
```

Comment:
@samedii does this work with all of our workflow? I vaguely recall github referenced packages to crash with something (CI maybe)?

Reply:
Maybe we had issues with pip

Reply:
So we can assume it's probably ok to do this now then?